examples/tensorflow/classification/main.py

```diff
@@ -21,7 +21,7 @@
 from nncf.config.utils import is_accuracy_aware_training
 from nncf.tensorflow.helpers.model_creation import create_compressed_model
 from nncf.tensorflow import create_compression_callbacks
-from nncf.tensorflow.helpers.model_manager import TFOriginalModelManager
+from nncf.tensorflow.helpers.model_manager import model_manager
```

Comment:
Suggest to somehow specify, that this manager is TF specific. For example, factory named `TFModelManager`

Reply:
Thanks for the comment! I fixed it.